charts/access/ms-teams/README.md

```diff
@@ -0,0 +1,69 @@
+# Teleport Access Request MsTeams Plugin
+
+This chart sets up and configures a Deployment for the Access Request MsTeams plugin.
+
+## Installation
+
+### Prerequisites
+
+As the MsTeams setup requires to download the plugin locally to generate assets to load in MsTeams,
+you must follow [the MsTeams access request guide](https://goteleport.com/docs/access-controls/access-request-plugins/ssh-approval-ms-teams/).
+When generating the identity file, choose the "proxy" solution that generates a single file `auth.pem`.
```

Comment:
:nail_care:
At first it was not clear what you meant by "proxy" solution.
Maybe we can add here the full Tab title? `Connect to the Proxy Service`?

Reply:
Done